app/addons/replication/assets/less/replication.less

```diff
@@ -11,187 +11,102 @@
-      background-color: @brandPrimary;
-      color: #fff;
-    }
+  input, select {
+    font-size: 14px;
+  }
+  input {
+    width: 246px;
+  }
+  select {
+    width: 246px;
+    margin-bottom: 10px;
+    background-color: white;
+    border: 1px solid #cccccc;
+  }
+  .styled-select {
+    width: 250px;
   }
 
-
-  .options {
-    position: relative;
-    &:after {
-      content: '';
-      display: block;
-      position: absolute;
-      right: -16px;
-      top: 9px;
-      width: 0;
-      height: 0;
-      border-left: 5px solid transparent;
-      border-right: 5px solid transparent;
-      border-bottom: 5px solid black;
-      border-top: none;
-    }
-    &.off {
-      &:after {
-      content: '';
-      display: block;
-      position: absolute;
-      right: -16px;
-      top: 9px;
-      width: 0;
-      height: 0;
-      border-left: 5px solid transparent;
-      border-right: 5px solid transparent;
-      border-bottom: none;
-      border-top: 5px solid black;
-      }
-    }
+  .span3 {
+    text-align: right;
+    margin-top: 12px;
   }
-  .control-group {
-    label {
-      float: left;
-      min-height: 30px;
-      vertical-align: top;
-      padding-right: 5px;
-      min-width: 130px;
-      padding-left: 0px;
-    }
-    input[type=radio],
-    input[type=checkbox] {
-      margin: 0 0 2px 0;
+  .remote-connection-details {
+    margin: 15px 0;
+  }
+  .connection-url {
+    width: 100%;
+  }
+  .buttons-row {
+    margin-top: 10px;
+    a {
+      padding: 12px;
     }
   }
+  .typeahead {
+    width: 100%;
+  }
 
-  .circle {
-    z-index: 0;
-    position: absolute;
-    top: 20px;
-    left: 15px;
-
-    &:after {
-      width: 70px;
-      height: 70px;
-      content: '';
-      display: block;
-      position: relative;
-      margin: 0 auto;
-      border: 1px solid @greyBrownLighter;
-      -webkit-border-radius: 40px;
-      -moz-border-radius: 40px;
-      border-radius:40px;
-    }
+  hr {
+    margin: 6px 0 15px;
+  }
+  .section-header {
+    font-weight: bold;
+    font-size: 14pt;
   }
-  .swap {
-    text-decoration: none;
-    z-index: 30;
+}
+
+#dashboard-content .replication-page {
+  padding-top: 25px;
+}
+
+.connection-url-example {
+  font-size: 9pt;
+  color: #999999;
+  margin-bottom: 8px;
+}
+
+.custom-id-field {
+  position: relative;
+  width: 250px;
+
+  span.fonticon {
     cursor: pointer;
     position: absolute;
-    font-size: 40px;
-    width: 27px;
-    height: 12px;
-    top: 31px;
-    left: 30px;
+    right: 6px;
+    top: 8px;
+    font-size: 11px;
+    padding: 8px;
+    color: #999999;
+    .transition(all 0.25s linear);
     &:hover {
-      color: @greyBrownLighter;
+      color: #333333;
+    }
+    input {
+      padding-right: 32px;
     }
   }
 }
 
-#replicationStatus {
-  &.showHeader {
-    li.header {
-      display: block;
-      border: none;
-    }
-    ul {
-      border:1px solid @greyBrownLighter;
-    }
+
+body .Select div.Select-control {
```

Comment:
Awesome thanks Michelle. I'll make the changes.

On 16 August 2016 at 05:04, Michelle Phung <notifications@github.com> wrote:

> In app/addons/replication/assets/less/replication.less
> <https://github.com/apache/couchdb-fauxton/pull/761#discussion_r74870084>:
>
> >      }
> >    }
> >  }
> >
> > -#replicationStatus {
> > -  &.showHeader {
> > -    li.header {
> > -      display: block;
> > -      border: none;
> > -    }
> > -    ul {
> > -      border:1px solid @greyBrownLighter;
> > -    }
> > +
> > +body .Select div.Select-control {
>
> OK i did it in a gist, since i don't know how to add to this this pr. You
> can copy and paste it:
>
> https://gist.github.com/michellephung/71b8658d38f37d67331c240e366fbf3b
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/couchdb-fauxton/pull/761/files/ed20c2db34e4b69515fd69e02c25531035e68d91#r74870084>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAK9AvjU3VYFG9yBTRw_7IdL1AJfDVt7ks5qgSiggaJpZM4JkZCx>
> .
>